src/Handler/Http/Middleware/SentryTrap/EnvelopeParser.php

```diff
@@ -75,7 +74,8 @@ private static function parseItem(StreamInterface $stream): EnvelopeItem
 
     /**
      * @param positive-int $possibleBytes Maximum number of bytes to read. If the read fragment is longer than this
-     *        an exception will be thrown. Default is 10MB
+     *                                    an exception will be thrown. Default is 10MB
+     *
```

Comment:
We align comments with the word `@param`

Reply:
This rule cannot be changed, only left or vertical